buildah.spec.rpkg

```diff
@@ -11,17 +11,19 @@
 
 %global with_debug 1
 
+# RHEL 8's default %%gobuild macro doesn't account for the BUILDTAGS variable, so we
```

Comment:
Soft suggestion:
```suggestion
# Starting with RHEL 8, the default %%gobuild macro doesn't account for the BUILDTAGS variable, so we
```

Reply:
Not quite `starting with rhel8`. But only rhel8.